tcf_website/management/commands/fix_multiple_objects_returned.py

```diff
@@ -0,0 +1,113 @@
+from functools import reduce
+from operator import or_
+
+from django.core.management.base import BaseCommand
+from django.db.models import Case, Count, F, FloatField, Q, Sum, Value, When
+
+from tcf_website.models import CourseInstructorGrade
+
+
+class Command(BaseCommand):
+    """This fixes existing `MultipleObjectsReturned` errors"""
+
+    help = 'Fix `MultipleObjectsReturned` errors that were caused by inconsistent email data'
+
+    def handle(self, *args, **options):
+        self.verbosity = options['verbosity']
+
+        # Number of cases we want to fix
+        qs = CourseInstructorGrade.objects\

```

Comment:
I assume `qs` stands for QuerySet, in which case we might want to change this variable name to something like `erroneous_instances` that conveys more information as to what it is.

Reply:
Done in f45bd0586dca0e7cfc008ddf272e194f660bbc75